docs/rvv-enviroment.md

```diff
@@ -0,0 +1,35 @@
+# Setting up enviroment for testing MLIR RVV dialect
+
+This guide will help to set up enviroment for testing RISC-V Vector Extention using buddy-mlir project and 
+corresponding RVV Dialect. As a target platform QEMU emulator is used.
+
+## Requirements
+
+Before proceed any further make sure that you installed dependecies  below
+
+* [LLVM dependecies](https://llvm.org/docs/GettingStarted.html#requirements)
+* [GNU Toolchain dependecies](https://llvm.org/docs/GettingStarted.html#requirements)
```

Comment:
Please change the `requirements` link for GNU toolchain to 
https://github.com/riscv-collab/riscv-gnu-toolchain#prerequisites